frontend/src/metabase/actions/containers/ActionCreator/FormCreator/FormCreator.styled.tsx

```diff
@@ -83,3 +83,11 @@ export const TopRightIcon = styled(Icon)`
   top: 0;
   right: 0;
 `;
+
+export const WarningBanner = styled.div`
+  padding: ${space(2)};
+  border: 1px solid ${color("accent4")};
```

Comment:
please change this color to `warning` too. Since `accent4` can be changed in appearance settings.

Reply:
missed it, done!